config/src/test/java/com/alibaba/nacos/config/server/service/repository/embedded/EmbeddedConfigInfoAggrPersistServiceImplTest.java

```diff
@@ -201,7 +201,7 @@ public void testFindConfigInfoAggrByPage() {
         String dataId = "dataId111";
         String group = "group";
         String tenant = "tenant";
-        
+
```

Comment:
Please don't change the indent, use nacos code style to reformat code.

Reply:
done